tests/bin/httpserver.php

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/php
+<?php
+echo "init...\n";
+require_once __DIR__.'/../../vendor/autoload.php';
+$cli_param = getopt('', ['config:',]);
+
+/** @var \NokitaKaze\TestHTTPServer\ServerSettings $options */
+$options = unserialize(file_get_contents($cli_param['config']));
+
+$options->onRequest = function ($server, $connect) {
+    /** @var \NokitaKaze\TestHTTPServer\Server $server */
+    /** @var \NokitaKaze\TestHTTPServer\ClientDatum $connect */
+    $output_filename = $server->get_option('output_filename');
+
+    $connect->server = null;
+    $saved = file_put_contents(
+        $output_filename, serialize(
+        [
+            'connection' => $connect,
+        ]
+    ), LOCK_EX
+    );
+    if ($saved === false) {
+        $server->answer($connect, 500, 'OK', '');
+        $server->close_connection($connect);
+        echo "can not save content\n";
+        exit(1);
+    }
+
+    $server->answer($connect, 200, 'OK', json_encode(['event' => uniqid()]));
+    $server->close_connection($connect);
+    echo "done\n";
+    exit(0);
+};
+$server = new \NokitaKaze\TestHTTPServer\Server($options);
+
+echo "listen...\n";
```

Comment:
ooops. Wrong async behavior. I need to add `init_listening`. There could be smallest chance to not open port before Sentry Client connects to it